documents/invoice/model.go

```diff
@@ -264,16 +262,16 @@ func (i *Invoice) getInvoiceSalts(invoiceData *invoicepb.InvoiceData) (*proofs.S
 // ID returns document identifier.
 // Note: this is not a unique identifier for each version of the document.
 func (i *Invoice) ID() ([]byte, error) {
-	coreDoc, err := i.PackCoreDocument()
+	coreDocModel, err := i.PackCoreDocument()
```

Comment:
Why do we need to do this here? Can't we expose a getter on the CoreDocumentModel to get the identifier? This seems a fairly standard thing.